### PR TITLE
Update Android instructions

### PR DESCRIPTION
**Pull request by cbhill**
This simplifies the Android instructions. We don't want to encourage users to use Volley since we are moving to Retrofit, but I think it's fine to let the candidate chose the library on their own. Also, we don't need to mention which Android version we will be testing on. We can leave it up to them to set the minimum SDK version and ask them to explain why if necessary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,9 +35,9 @@ We’ll be looking for clean, well-structured code and a polished user interface
 
 ##### Android
 
-Use Java with standard Android XML layouts. You can use any libraries and tools you like. Volley and Glide are not required but recommended. 
+Use Java with standard Android XML layouts. You can use any libraries and tools you like, but be prepared to explain why.
```
Comment (jeffreyzupka) [inline README.md]: We are explicitly asking for a README in the web instructions. How do you feel about doing the same here?

**Comment by jeffreyzupka** [comment]
:thumbsup:

File: README.md (hunk 1 above)
Comment (cbhill) [inline README.md]: Good suggestion, done

**Comment by snadjafi** [comment]
:+1:

**Pull request merged**